### ISSUE TITLE
Masks in the bank disappear after the timer ends

### ISSUE THREAD

**Issue by nohh** [WarEmu/WarBugs#20544]
Masks in the bank disappear after the timer ends.
The masks in the bag are successfully converted after the timer ends.

**Steps to reproduce the problem:**
1. Put mask in bank
2. Wait while the timer ends
3. Mask will be converted and after second disappears

![mask_decay](https://user-images.githubusercontent.com/19315739/201497126-89e39e1b-4f5c-4197-8c8f-79bc6b6663d4.png)

**Comment by Rebiff** [comment]
did they not just get sent to your bags instead from the bank? thats what happened to me

**Comment by waerter** [comment]
No they didnt, They decay in there permanent form and then decay completely.
This is very unfortunate because especially the joyous visage mask was chard to come by at order side. 
I you need a verification:
![Mask decayed](https://user-images.githubusercontent.com/62266826/201517656-b748fc96-8ac9-44ef-84a6-7fbf25be1c8e.jpg)

**Comment by Nondrum** [comment]
I am having the same issues.  Some of the masks from my Zeal, DoK, and BG are gone.  I have tried to put in multiple GM appeals, but they told me I needed to add something to the bugtracker.
![skull_mask-gone](https://user-images.githubusercontent.com/118139763/201550047-8ac14432-46d2-4e12-8c21-5bfa4d189592.png)

**Comment by Pepparn97** [comment]
Yeah this happened to me too, lost three masks on a  dok named "Cherubiel". Is there anyway to get them back?

**Comment by Uberlix** [comment]
To add to this, i lost all the masks i gathered during the Event on my Choppa "Rukzukzahnluk" as well. They were all in the bank, and are now gone. The Masks on my other characters are fine, because they are in their respective inventory.
